fenicsprecice/fenicsprecice.py

```diff
@@ -1,15 +1,19 @@
 """
-Adapter module to provide an API to simulate partitioned coupled problems with FEniCS as a coupling participant
+Adapter to FEniCS solver handles CustomExpression and initialization of the FEniCS adapter.
+:raise ImportError: if PRECICE_ROOT is not defined
 """
 import numpy as np
 from .config import Config
 import logging
 import precice
-from .adapter_core import FunctionType, CouplingMode, determine_function_type, convert_fenics_to_precice, \
-    get_coupling_boundary_vertices, get_coupling_boundary_edges, get_forces_as_point_sources
-from .expression_core import SegregatedRBFInterpolationExpression
+from .adapter_core import FunctionType, determine_function_type, convert_fenics_to_precice, \
+    get_fenics_coupling_boundary_vertices, get_owned_coupling_boundary_vertices, get_unowned_coupling_boundary_vertices, \
+    get_coupling_boundary_edges, get_forces_as_point_sources, determine_shared_vertices, communicate_shared_vertices,\
+    CouplingMode
+from .expression_core import SegregatedRBFInterpolationExpression, EmptyExpression
 from .solverstate import SolverState
 from fenics import Function, FunctionSpace
+from mpi4py import MPI
```

Comment:
Why do we use `from mpi4py import MPI` here and `from fenics import MPI` [above](https://github.com/precice/fenics-adapter/pull/71/files#diff-28d189e544b09e294873b3e88f20eb28e6fba8b856511159660c65782292e524R10)? Can we make this consistent either way or the other?